src/composer/src/build.rs

```diff
@@ -326,12 +328,14 @@ fn kern_gen_make_cmd(input_constructor: &String, kern_output: &String, _s: &Syst
 
 pub struct DefaultBuilder {
     builddir: String,
+    rebuildflag: Option<String>,
```

Comment:
I don't think this makes much sense. You're using this as either `None` or `Some("REBUILD")`. As there is only one option for rebuilding, shouldn't this just be a `bool`?

If you're trying to make it so that we can rebuild in different ways in the future, I wouldn't worry about it for now. Keep it simple. Later would make it an `enum` (or more likely, a vec of an enum).

Reply:
change the type to bool